tests/OtpTest.php

```diff
@@ -120,4 +120,45 @@ public function testHotpInvalidCounter()
 		$this->Otp->hotp($this->secret, 'a');
 	}
 
+	/**
+	 * Tests Otp->checkHotpResync()
+	 */
+	public function testHotpResync()
+	{
+		$secret = $this->secret;
+
+		// test default counter window
+		$this->assertEquals(0, $this->Otp->checkHotpResync($secret, 0, '755224'));
```

Comment:
I'd use "assertSame" here to make sure it's a pure int that gets returned in these cases, otherwise thhere might be some type juggling for the first two.